.github/workflows/container.yml

```diff
@@ -266,13 +275,28 @@ jobs:
       - name: Check out the code
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
-      - name: Set up Go
-        uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
+      - name: Install devbox
+        uses: jetpack-io/devbox-install-action@v0.7.0
```

Comment:
It might be possible to remove this action if `direnv` uses the Nix flake

Would still require to install Nix tho, [cachix/install-nix-action](https://github.com/cachix/install-nix-action) could do that